enterprise/api_client/enterprise.py

```diff
@@ -0,0 +1,50 @@
+
+# -*- coding: utf-8 -*-
+"""
+Client for communicating with the Enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+from django.core.cache import cache
+
+from enterprise.api_client.lms import LmsApiClient
+from enterprise.utils import get_cache_key, traverse_pagination
+
+
+class EnterpriseApiClient(LmsApiClient):
```

Comment:
I agree. We can cleanup the clients for different services by getting the authentication token form LMS then using `EdxRestApiClient` module for accessing the API's. We can have a separate story for this.